modules/gapi/src/backends/fluid/gfluidbackend.hpp

```diff
@@ -153,6 +153,8 @@ class GFluidExecutable final: public GIslandExecutable
     virtual void run(std::vector<InObj>  &&input_objs,
                      std::vector<OutObj> &&output_objs) override;
 
+    using GIslandExecutable::run; // (IInput&, IOutput&) version
```

Comment:
Is this required?

Reply:
Yes, there were warnings on android and mac that another (non virtial) run below hides this method

Reply:
ok I see now. Thanks!